quick-start/index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link type="image/png" rel="icon" href="https://raw.githubusercontent.com/GMartigny/pencil.js/master/media/mini-logo.png">
+    <title>Quick-start - Pencil.js</title>
+    <link type="text/css" rel="stylesheet" href="../res/reset.css"/>
+    <link type="text/css" rel="stylesheet" href="../res/style.css"/>
+</head>
+<body>
+<div class="title-screen">
+    <div class="main-title">
+        <h1>
+            Quick-start
+        </h1>
+        <small>
+            Clear step-by-step demonstration of Pencil.js capabilities
```

Comment:
```suggestion
            A clear step-by-step demonstration of Pencil.js
```
```suggestion
            A clear step-by-step demonstration of the capabilities of Pencil.js
```

Reply:
> **of** the capabilities **of** Pencil.js

Two "of" feel a little heavy, don't you think ?

Reply:
Yes, the double "of" is why I'd made the first suggestion, questioning if "capabilities" is even adding information in the context of a demonstration.